frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -176,7 +176,9 @@ export function ChoroplethMap(props: ChoroplethMapProps) {
       projections: [
         {
           name: US_PROJECTION,
-          type: "albersUsa",
+          type: TERRITORY_CODES.includes(props.fips.code)
```

Comment:
Can you use the isTerritory function here? ie something like Fips(props.fips.code).isTerritory()

Reply:
o yaaa! nice